### DISCUSSION TITLE
Allow templating of URL parameters in the Webhook Contact Point

### DISCUSSION THREAD

**Discussion by joeblubaugh** [grafana/grafana#49973] [Alerting]
This discussion originates from this issue: #47102

The user would like to 
A: Be able to set URL parameters with a key-value interface, instead of writing them directly into the `URL` field.
B: Be able to template the values of those URL parameters based on https://grafana.com/docs/grafana/latest/alerting/unified-alerting/message-templating/template-data/

This would allow, for example, a URL like:
`https://myalerts.com?title=the%20system%20is%20down%21`

**Comment by santihernandezc** [comment]
@joeblubaugh I just merged a [PR to add templates to webhook URLs](https://github.com/grafana/grafana/pull/57296)

**Comment by Ayatallah** [comment]
Do we have this feature yet? 
To be able to have url for grafana webhook contact point with parameter passed to it? @santihernandezc @joeblubaugh

**Reply by Ayatallah** [reply]
and is this feature in the grafana community edition or enterprise edition?

**Reply by armandgrillet** [reply]
This is in Grafana community edition since 9.3.
